ratis-client/src/main/java/org/apache/ratis/client/impl/OrderedStreamAsync.java

```diff
@@ -43,8 +43,8 @@
     private final long seqNum;
     private final CompletableFuture<DataStreamReply> replyFuture = new CompletableFuture<>();
 
-    DataStreamWindowRequest(long streamId, long offset, ByteBuffer data, long seqNum, Type type){
-      super(streamId, offset, data, type);
+    DataStreamWindowRequest(Type type, long streamId, long offset, ByteBuffer data, long seqNum) {
```

Comment:
Why choose to reorder parameters? if don't do so, this PR will be smaller?

Reply:
I am ok though if the answer is just personal preference :) just want to make sure I am not missing something important.

Reply:
`Type type, long streamId, long offset` should be put together in the same order.  It will be consistent throughout the code.